packages/newspringchurchapp/src/content-single/Features/SermonNotes.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  H3,
+  PaddedView,
+  ActionCard,
+  Touchable,
+  Icon,
+} from '@apollosproject/ui-kit';
+import share from '../../utils/content/share';
+
+const SermonNotes = ({ contentId, features }) => {
+  const [sharedMsg, changeSharedMsg] = useState('');
+  const [enhancedFeatures, enhanceFeatures] = useState([]);
```

Comment:
Is there any way we could differentiate these names a little more?

Reply:
What do you have in mind?